docs/initial-milestones.md

```diff
@@ -0,0 +1,65 @@
+# OpenTelemetry: A Roadmap to Convergence
+
+This document covers the intial milestones for each SIG developing an implementation of the OpenTelemetry Specification. For each new language, we want to quickly achieve parity with existing OpenTracing and OpenCensus implementations.
```

Comment:
I'd suggest to use language instead of SIG. We have Agent/Collector SIG for which this doc may not be applicable.

Reply:
```suggestion
This document covers the intial milestones for each language repository developing an implementation of the OpenTelemetry Specification. For each new language, we want to quickly achieve parity with existing OpenTracing and OpenCensus implementations.
```

Reply:
Or can state at the top of the doc that SIGs in this doc are scoped to language libraries, and do not include agent/collector